plugins/modules/system/sysupgrade.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2020, Andrew Klaus <andrewklaus@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: sysupgrade
+
+short_description: Manage OpenBSD system upgrades
+
+version_added: "2.10"
+
+description:
+    - This module can be used to manage OpenBSD system upgrades using sysupgrade.
+
+options:
+    snapshot:
+        description:
+            - Apply the latest snapshot.
+            - Otherwise release will be applied.
+        default: false
+        required: false
+        type: bool
+    force:
+        description:
+            - Force upgrade (for snapshots only).
+        required: false
+        type: bool
+    keep_files:
+        description:
+            - Keep the files in /home/_sysupgrade.
+            - By default, the files will be deleted after the upgrade.
+        required: false
+        type: bool
+    fetch_only:
+        description:
+            - Fetch and verify files and create /bsd.upgrade but do not reboot.
+        default: true
+        required: false
+        type: bool
+
+author:
+    - Andrew Klaus (@precurse)
+'''
+
+EXAMPLES = r'''
+- name: Upgrade to latest release
+  sysupgrade:
+  become: true
```

Comment:
```suggestion
```
it doesn't relate to the module use directly (imo, it's a question how to use ansible in general). Just a recommendation how to make examples clearer

Reply:
Makes sense to me